build.rs

```diff
@@ -7,55 +7,71 @@ use rusoto_codegen::{Service, generate};
 
 /*
 gamelift/2015-10-01/service-2.json:    "protocol":"json"
-
-storagegateway//service-2.json:    "protocol":"json",
 support/2013-04-15/service-2.json:    "protocol":"json"
-swf/2012-01-25/service-2.json:    "protocol":"json"
-waf/2015-08-24/service-2.json:    "protocol":"json",
-workspaces/2015-04-08/service-2.json:    "protocol":"json"*/
+*/
+
+// expand to use cfg!() so codegen only gets run for services
+// in the feautres list
```

Comment:
Yeah, it will.  I noticed that my builds were taking an absurdly long time, even for what should have only been a single service.  This fixed it.